test/access-token.test.js

```diff
@@ -351,10 +361,33 @@ function createTestApp(testToken, settings, done) {
   var appSettings = settings.app || {};
   var modelSettings = settings.model || {};
 
+  var tokenOptions = {};
+  var tokenKeySettings = settings.tokenKey || {};
+  var headers = tokenKeySettings.headers || [];
+  var cookies = tokenKeySettings.cookies || [];
+  var params = tokenKeySettings.params || [];
+  var defaultTokenKeys = tokenKeySettings.defaultTokenKeys || undefined;
+
+  if (defaultTokenKeys === undefined) {
+    tokenOptions = {
+      model: Token,
+      currentUserLiteral: 'me'
+    };
+  } else {
+    tokenOptions = {
+      model: Token,
+      currentUserLiteral: 'me',
+      defaultTokenKeys: defaultTokenKeys,
+      headers: headers,
+      cookies: cookies,
+      params: params
+    };
+  }
```

Comment:
This looks overcomplicated to me. How about the following solution?

```js
var tokenOptions = settings.token || {};
tokenOptions.model = Token;
tokenOptions.currentUserLiteral = 'me';
```

Usage:

```js
createTestAppAndRequest(this.token,
  { token: { headers: ['authorization'], defaultTokenKeys: false } },
  done)
```